src/com/nutrons/stronghold/Drivetrain.java

```diff
@@ -0,0 +1,29 @@
+package com.nutrons.stronghold;
+
+import com.nutrons.framework.Subsystem;
+import com.nutrons.framework.controllers.ControllerEvent;
+import com.nutrons.framework.controllers.RunAtPowerEvent;
+import com.nutrons.framework.util.FlowOperators;
+import io.reactivex.Flowable;
+import io.reactivex.functions.Consumer;
+
+class Drivetrain implements Subsystem {
+  private final Flowable<ControllerEvent> y1;
```

Comment:
What are `y1` and `y2`.  I'm assuming they're joystick streams, but be on the lookout for names that aren't obvious.  Alternately, add a comment.